CHANGELOG.md

```diff
@@ -1,12 +1,15 @@
 <!--lint disable maximum-line-length-->
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [v1.1.0-beta.1] - 2024-08-10
```

Comment:
Listed as 2024-08-05 in the citation?

```suggestion
## [v1.1.0-beta.1] - 2024-08-05
```

Reply:
Needs to be updated once we do the actual release...